distribution/osbuild-worker.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
There is already a service file in the init/ directory. Please delete that as this one replaces it. I would copy some of that logic though, it is a service templace, so people can enable as many or as few workers as they want (but by default only one is enabled when you call "systemctl enable osbuild-worker@.service").